packages/core/addon/components/navi-table-sort-icon.hbs

```diff
@@ -1,5 +1,5 @@
 {{!-- Copyright 2021, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-<span class="navi-table-sort-icon {{this.sortClass}}" ...attributes>
-  <NaviIcon @icon="sort-asc" class="navi-table-sort-icon__caret sort-icon__caret-up" />
-  <NaviIcon @icon="sort-desc" class="navi-table-sort-icon__caret sort-icon__caret-down" />
+<span class="navi-table-sort-icon {{this.sortClass}} flex flex-column p-l-3" ...attributes>
+  <DenaliIcon @icon="arrowhead-up" @size="extrasmall" class="navi-table-sort-icon__caret navi-table-sort-icon__caret-up" />
+  <DenaliIcon @icon="arrowhead-down" @size="extrasmall" class="navi-table-sort-icon__caret navi-table-sort-icon__caret-down" />
```

Comment:
The design team suggested using a solid arrow. I believe this will do it:
```suggestion
  <DenaliIcon @icon="arrowhead-up-solid" @size="extrasmall" class="navi-table-sort-icon__caret navi-table-sort-icon__caret-up" />
  <DenaliIcon @icon="arrowhead-down-solid" @size="extrasmall" class="navi-table-sort-icon__caret navi-table-sort-icon__caret-down" />
```